packages/astro/src/core/build/static-build.ts

```diff
@@ -255,6 +255,8 @@ async function ssrBuild(
 							return 'renderers.mjs';
 						} else if (chunkInfo.facadeModuleId === RESOLVED_SSR_MANIFEST_VIRTUAL_MODULE_ID) {
 							return 'manifest_[hash].mjs';
+						} else if (chunkInfo.facadeModuleId === settings.adapter?.serverEntrypoint) {
+							return 'adapter_[hash].mjs';
```

Comment:
What's this change about?

Reply:
This changes depends on `globalThis.crypto` which isn't available in Node 18. We have a polyfill for it, but currently the manifest is created before the adapter runs, so the adapter isn't able to polyfill it in time.

So the fix was to make the adapter an entrypoint, causing it to be built to a separate file and therefore it would run before the manifest. This line here just names it a certain way to make debugging easier.